ql/ql/src/codeql_ql/StructuredLogs.qll

```diff
@@ -213,14 +213,43 @@ module KindPredicatesLog {
         millisecond = s.regexpCapture(r, 7).toInt()
       )
     }
+
+    int getResultSize() { result = this.getNumber("resultSize") }
   }
 
   class SentinelEmpty extends SummaryEvent {
     SentinelEmpty() { evaluationStrategy = "SENTINEL_EMPTY" }
   }
 
+  class PipeLineRun extends Object {
+    PipeLineRuns runs;
+    int index;
+
+    PipeLineRun() { runs.getObject(index) = this }
+
+    PipeLineRuns getArray() { result = runs }
+
+    string getRaReference() { result = this.getString("raReference") }
```

Comment:
This one also goes against the style guide.

Reply:
Thanks! Fixed in f155f19725bfe9930796f9da24b743afdf111f92.